sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -605,6 +606,27 @@ def _to_internal_entity(self):
 
 
 class CorrelationRuleFilter(object):
+    """Represents the correlation filter expression.
+
+    :param correlation_id: Identifier of the correlation.
+    :type correlation_id: str
+    :param message_id: Identifier of the message.
+    :type message_id: str
+    :param to: Address to send to.
+    :type to: str
+    :param reply_to: Address of the queue to reply to.
+    :type reply_to: str
+    :param label: Application specific label.
+    :type label: str
+    :param session_id: Session identifier.
+    :type session_id: str
+    :param reply_to_session_id: Session identifier to reply to.
+    :type reply_to_session_id: str
+    :param content_type: Content type of the message.
+    :type content_type: str
+    :param properties: dictionary object for custom filters
+    :type properties: dict[str, str]
```

Comment:
dict[str,Any]?
```suggestion
    :type properties: dict[str, Any]
```

Reply:
I changed it to dict[str, Union[str, int, float, bool, datetime, timedelta]], same as parameters of sql rule filter.